lib/contextMenu.js

```diff
@@ -538,6 +538,7 @@ export default function (self) {
       });
     }
   }
+  //#region addDefaultContextMenuItem
```

Comment:
Does the `//#region` mean anything? I'm not familiar with the notation

Reply:
This notation is for the editor. Because I think some code fragments that I added are so long. So I add this notation for code folding, like this:

<img width="492" alt="Screenshot 2022-01-06 at 00 21 44" src="https://user-images.githubusercontent.com/4303130/148251943-eade9d68-f39d-4449-8721-54b64ab22c0a.png">

This style of code folding seems from the Visual Studio. And I am using the VSCode now, which supports this style. But I am not sure whether other editors can support this style.

Reply:
Ah I see, cool! I'm also on VS Code, but I didn't know about this feature yet, thanks!